modules/sff/module/inc/sff/8472.h

```diff
@@ -109,6 +109,7 @@
 #define SFF8472_CC6_CBE_BASE_LX           0x10
 #define SFF8472_CC6_CBE_BASE_FX           0x20
 #define SFF8472_CC6_BASE_BX10             0x40
+#define SFF8472_CC6_BASE_BX40             0x40
```

Comment:
These have the same value, since the following code will test for `BX10` first, every module with this bit set will be identified as `BX10`, and never as `BX40`.